src/hooks.ts

```diff
@@ -0,0 +1,141 @@
+/**
+ * Copyright 2020, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { useCallback, useContext, useEffect, useState } from 'react';
+import * as optimizely from '@optimizely/optimizely-sdk';
+import { getLogger } from '@optimizely/js-sdk-logging';
+
+import { setupAutoUpdateListeners } from './autoUpdate';
+import { VariableValuesObject, OnReadyResult } from './client'
+import { OptimizelyContext } from './Context';
+
+const useFeatureLogger = getLogger('useFeature');
```

Comment:
Does this just drop this string in the log prefix? Not able to find the `js-sdk-logging` repo and am unsure how it works.

Reply:
Yup! It's used as namespace in the log manager (as well as a logging prefix for that instance).

The logger package lives inside the javascript-sdk: https://github.com/optimizely/javascript-sdk/blob/master/packages/logging/src/logger.ts

Reply:
Ah, another js-sdk package. Shoulda looked there.